packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -26,6 +26,7 @@ class BuildWebCommand extends BuildSubCommand {
     usesWebRendererOption();
     addEnableExperimentation(hide: !verboseHelp);
     addNullSafetyModeOptions(hide: !verboseHelp);
+    addNativeNullAssertions(hide: false);
```

Comment:
Why is this not hidden but `run` is?

Reply:
For build web, it makes sense 100% of the time. For run, it is only applicable to web targets so I hid it under the verbose options